LFS171x/fabric-material/basic-network/docker-compose.yml

```diff
@@ -0,0 +1,123 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '2'
+
+networks:
+  basic:
+
+services:
+  ca.example.com:
+    image: hyperledger/fabric-ca:x86_64-1.0.3
```

Comment:
Do we need the specific label on the individual images?

Reply:
I personally was getting errors that it couldn't find fabric-ca latest.

Reply:
Each of the images in this file has a version number. By including the version number here, we limit people from being able to use later versions released by the Hyperledger Fabric community.